com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md

```diff
@@ -13,7 +16,7 @@ Resolve only exports LUTs of size 33 so set **Grading LUT Size** to **33** for i
 Export the current [Camera](HDRP-Camera.md)'s view to a log-encoded EXR file. To do this:
 
 1. In the Scene view or Hierarchy, select a GameObject that contains a Camera component.
-2. Select **Edit** > **Rendering** > **Render Selected HDRP Camera to Log EXR** **(**or press **Ctrl+Shift+Alt+E)**.
```

Comment:
any reasons to remove the shortcut here? this is useful for people

Reply:
hmm I notice doc is very simialr to: https://github.com/Unity-Technologies/Graphics/pull/6976 . Not sure why we have two page here, we should share stuff

Reply:
About the shortcut, I thought about removing it because it is windows specific but I actually agree with you that it makes more sense to leave it. About the similar pages, yes they are very similar but cover different programs. @Vic-Cooper do you think they should stay as two pages or be merged?